src/main/java/org/jenkinsci/plugins/docker/commons/impl/RegistryKeyMaterialFactory.java

```diff
@@ -112,6 +114,34 @@ public KeyMaterial materialize() throws IOException, InterruptedException {
         return new RegistryKeyMaterial(dockerConfig, new EnvVars("DOCKER_CONFIG", dockerConfig.getRemote()));
     }
 
+    /**
+     * Copy docker config source data to another docker config
+     * @param dockerConfig
```

Comment:
Please do not refer parameters in Javadoc if you do not actually define documentation. It leads to warnings during the build